### DISCUSSION TITLE
Precipitation Notification

### DISCUSSION THREAD

**Discussion by nrestell** [breezy-weather/breezy-weather#335] [Q&A] [answered]
Do any of the sources offer precipitation notification (e.g., rain in 32 minutes)? It is not clear from the help (there are "Precipitations in the next hour" and "Alerts" but not "precipitation alerts"). Could this be something that is implemented? Other than this feature - I am loving this app! :)

**Comment by papjul** [comment]
Alerts are usually, if not all the time, emitted by government meteorological institutions, so some countries might provide this kind of alert for heavy rain conditions, others not.

Currently, the Precipitation notification only check once a day (during the first background update job the day) if there will be precipitation during the day, and also if there will in the short term (4 hours I think) and will send you a notification in that case, if you enabled that notification. I kept this behavior as it was in Geometric Weather, so if you have any idea on how to improve it, I’m open to hear it.

**Reply by papjul** [reply]
I think for sources that provide minutely forecast I could make this "Rain will start in 32 minutes" notification on background update job run.

We just have to decide:
1) a threshold (if we send a notification every time there is 0,1 mm expected, it’s really unhelpful)
2) what to do if it’s already raining (you probably already know that by looking at your window). Maybe something like "Rain expected to last for at least X minutes"?

Can I let you open a new idea on Issues with what you think would be best?

**Comment by papjul** [comment, marked as answer]
Follow-up: #336
